modules/account-lib/src/coin/hbar/transaction.ts

```diff
@@ -155,15 +170,15 @@ export class Transaction extends BaseTransaction {
         {
           address: txJson.to,
           value: txJson.amount,
-          coin: this._coinConfig.name,
+          coin: txJson.tokenId || this._coinConfig.name,
```

Comment:
should use display name here instead of tokenId if possible (use a map from tokenId -> tokenName).

Reply:
`outputs` is used in various places and we expect this to be a human readable value